### PR TITLE
test(integration): enable e2e

### PR DESCRIPTION
**Pull request by buenaflor**
#skip-changelog

### CODE REVIEW

File: flutter/example/integration_test/integration_test.dart [hunk 1]
```diff
@@ -162,66 +163,75 @@ void main() {
...
+    var output = find.byKey(const Key('output'));
+    late Fixture fixture;
+
+    setUp(() {
+      fixture = Fixture();
+    });
+
+    testWidgets('captureException', (tester) async {
+      late Uri uri;
+      await restoreFlutterOnErrorAfter(() async {
```
Comment (vaind) [inline flutter/example/integration_test/integration_test.dart:179]: Looks like something for setup/teardown functions?

Comment (buenaflor) [reply flutter/example/integration_test/integration_test.dart:179]: I think it needs to be restored before expect calls are made, wouldn't work in setup or teardown

I'll double check

Comment (vaind) [reply flutter/example/integration_test/integration_test.dart:179]: ah, if that's the case, could you please add a comment to the function so anyone coming back to this won't have the same idea as I've just had?

File: .github/workflows/flutter_test.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ on:
       - "flutter/**"
 
 env:
-  SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+  SENTRY_AUTH_TOKEN_E2E: ${{ secrets.SENTRY_AUTH_TOKEN_E2E }}
 
 jobs:
```
Comment (vaind) [inline .github/workflows/flutter_test.yml:18]: FYI it's now preferrable (faster) to run android emulator on linux: https://github.com/ReactiveCircus/android-emulator-runner#running-hardware-accelerated-emulators-on-linux-runners

Comment (buenaflor) [reply .github/workflows/flutter_test.yml:18]: oh yeah, will change it

Comment (buenaflor) [reply .github/workflows/flutter_test.yml:18]: oh yeah, will change it

File: .github/workflows/flutter_test.yml [hunk 3]
```diff
@@ -35,6 +35,12 @@ jobs:
...
         sdk: ["stable", "beta"]
     steps:
+      - name: Enable KVM group perms
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
+          
       - name: checkout
         uses: actions/checkout@v4
```
Comment (vaind) [inline .github/workflows/flutter_test.yml]: to fix CI

```suggestion
      - name: checkout
        uses: actions/checkout@v4

      - name: Enable KVM group perms
        run: |
          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
          sudo udevadm control --reload-rules
          sudo udevadm trigger --name-match=kvm
          
```

**Review by vaind — APPROVED** [review]
LGTM, hopefully it's going to be stable 🤞

**Comment by vaind** [comment]
CI doesn't work so I've changed this back to draft so it's not tempting anyone to merge (because of the approval)

**Comment by buenaflor** [comment]
yeah not sure why, emulator won't start for some reason after changing the runner to linux

**Comment by vaind** [comment]
> yeah not sure why, emulator won't start for some reason after changing the runner to linux

Works here, maybe you can find a difference
- https://github.com/getsentry/sentry-java/blob/main/.github/workflows/integration-tests-ui-critical.yml
- https://github.com/getsentry/sentry-dotnet/blob/main/.github/workflows/device-tests-android.yml

File: flutter/lib/src/screenshot/widget_filter.dart [hunk 4]
```diff
@@ -233,9 +233,11 @@ extension on Element {
 @internal
 extension Opaqueness on Color {
   @pragma('vm:prefer-inline')
+  // ignore: deprecated_member_use
   bool get isOpaque => alpha == 0xff;
 
   @pragma('vm:prefer-inline')
+  // ignore: deprecated_member_use
   Color asOpaque() => isOpaque ? this : Color.fromARGB(0xff, red, green, blue);
```
Comment (buenaflor) [inline flutter/lib/src/screenshot/widget_filter.dart:255]: @vaind analyzer complains, I'm ignoring this if you're fine with it

I guess it was deprecated in the new version

Comment (vaind) [reply flutter/lib/src/screenshot/widget_filter.dart:255]: It's ok, I'll have a look in a separate PR

**Review by vaind — APPROVED** [review]

**Pull request merged**